os/customizing-docker.md

```diff
@@ -95,13 +93,43 @@ cp -p ~/cfssl/client.pem cert.pem
 cp -p ~/cfssl/client-key.pem key.pem
 ```
 
-### Drop-in Configuration
+### Enable the Secure Remote API on a New Socket
+
+**NOTE:** Skip this part if you use CoreOS releases older than 949.0.0.
```

Comment:
You'll have to link out since Github doesn't have the different versions. Just link to the appropriate thing on https://coreos.com/os/docs/942.0.0/customizing-docker.html. Sound good?